crypto/fipsmodule/ec/ec_test.cc

```diff
@@ -2535,8 +2536,8 @@ static void openvpn_extkey_ec_finish(EC_KEY *ec)
 }
 
 TEST(ECTest, ECKEYMETHOD) {
-  EC_KEY *ec = EC_KEY_new();
-  ASSERT_TRUE(ec);
+  bssl::UniquePtr<EC_KEY> ec(EC_KEY_new());
```

Comment:
Huh, was this always leaked? I'm surprised Valgrind or Address Sanitizer didn't find this.